kuma/wiki/jinja2/wiki/react_document.html

```diff
@@ -1,68 +1,67 @@
-{% extends "wiki/react_base.html" %}
```

Comment:
This template was complicated and hard for me to understand, so I started with react_base and wiki/react_base and got rid of the extends calls and the block definitions. Then I cut things that were not needed for the static site and cut that parts of the template that were actually rendering document content, since React is taking that job.  What's left is mostly the document head, but also the footer at the bottom of every page and things like the newsletter and payments stuff that I still need to figure out. Big changes here, and probably many more changes still needed. But it seems to be in a reasonable working state for now.